slackbotExercise.py

```diff
@@ -95,6 +97,13 @@ def __str__(self):
     def __repr__(self):
         return 'Exercises("%s", "%s", "%s")' % (self.exercise, self.users, self.timestamp)
 
+class Reminder:
+    def __init__(self, exercise_timestamp, reminder_timestamp, userid, exercise):
+        self.exercise_timestamp = exercise_timestamp
+        self.reminder_timestamp = reminder_timestamp
```

Comment:
reminder_timestamp is being assigned as a datetime object where as exercise_timestamp is being assigned as a string (the string timestamp that we get back from slack). It's a little confusing and might be worth naming them differently.